features/Steps/MailSteps.php

```diff
@@ -28,4 +28,9 @@ private function removeUuidFilePattern(string $value): string
         $uuidFilePattern = '/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\.(pdf|xlsx|json)/i';
         return preg_replace($uuidFilePattern, '', $value);
     }
+
+    private function fixNewLines(string $value): string
```

Comment:
Can you make the name a bit clearer? Maybe `removeCarriageReturn` ? Can you also add a comment on why this is necessary? This'll help future developers when they're reading this code.